apps/dash-mapd-demo/README.md

```diff
@@ -6,7 +6,7 @@ This Dash app updates real-time visual analytics by constantly querying a remote
 
 ## About this dataset
 
-The public dataset used in this app, "flight_2008_7M", includes every single record of US airline flights known by the Bureau of Transportation Statistics in the year 2008. By default, it can be loaded from the `insert_sample_data` script during the Omnisci server installation. A [jupyter notebook](https://github.com/plotly/dash-mapD-demo/blob/dev/flights_data_clean.ipynb) has been included to showcase tabular structure and query results from this dataset.
+The public dataset used in this app, "flight_2008_10k", includes every single record of US airline flights known by the Bureau of Transportation Statistics in the year 2008. By default, it can be loaded from the `insert_sample_data` script during the Omnisci server installation. A [jupyter notebook](https://github.com/plotly/dash-mapD-demo/blob/dev/flights_data_clean.ipynb) has been included to showcase tabular structure and query results from this dataset.
```

Comment:
not blocking but since this dataset was cut does it still include every single record from 2008?

Reply:
The Jupyter Notebook points to an archived repository and the data in the notebook references the old dataset.. Can we update the link to this repo? (and add the updated notebook in this repo as well)

Reply:
Fixed.